language-support/java/bindings-rxjava/src/main/java/com/daml/ledger/rxjava/CommandSubmissionClient.java

```diff
@@ -15,6 +16,15 @@
 /** An RxJava version of {@link com.daml.ledger.api.v1.CommandSubmissionServiceGrpc} */
 public interface CommandSubmissionClient {
 
+  Single<Empty> submit(CommandsBuilder params);
+
+  /**
+   * Use {@link #submit(CommandsBuilder)} instead
+   *
+   * @deprecated
+   * @since 2.5
+   */
+  @Deprecated
```

Comment:
Is this javadoc adequate?

Reply:
As mentioned above, I would make it to 2.5. The worst that can happen is that this doesn't make the cut for 2.5 and we'll need to update the version numbers as a separate commit. I would also argue that we probably want to roll out these deprecations as a single unit in one version across this and the `TransactionClient`.